react-app/src/utils/crumbs.test.ts

```diff
@@ -0,0 +1,114 @@
+// crumbs.test.ts
+import { describe, it, expect} from 'vitest';
+import { Authority } from 'shared-types/authority';
+import { Action } from 'shared-types/actions'; 
+import {
+  getDashboardTabForAuthority,
+  detailsAndActionsCrumbs,
+  dashboardCrumb,
+  detailsCrumb,
+  actionCrumb,
+} from './crumbs';
+
+
+
+describe('crumbs', () => {
+    describe('getDashboardTabForAuthority', () => {
+        it('should return "spas" for CHIP SPA', () => {
+          const result = getDashboardTabForAuthority(Authority.CHIP_SPA);
+          expect(result).toBe('spas');
+        });
+      
+        it('should return "spas" for Medicaid SPA', () => {
+          const result = getDashboardTabForAuthority(Authority.MED_SPA);
+          expect(result).toBe('spas');
+        });
+      
+        it('should return "waivers" for 1915(b)', () => {
+          const result = getDashboardTabForAuthority(Authority["1915b"]);
+          expect(result).toBe('waivers');
+        });
+      
+        it('should return "waivers" for 1915(c)', () => {
+          const result = getDashboardTabForAuthority(Authority["1915c"]);
+          expect(result).toBe('waivers');
```

Comment:
`Authority` values coming from URL params will not be in lowercase (i.e. the `Authority` enum values). Honestly, it's the codebase's fault for the confusion since there are two competing standards, but play around with the `localhost` and `console.log`. You'll see what I mean

Reply:
will work on it tomorrow

Reply:
resolved after reverting crumbs.ts to the original file and I changed ton --> as Authority.